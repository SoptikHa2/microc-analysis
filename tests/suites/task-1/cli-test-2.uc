// stdin: 40 2

main() {
  var x;

  x = input;
  output x+1;

  x = input;
  output x+1;

  return 0;
}
