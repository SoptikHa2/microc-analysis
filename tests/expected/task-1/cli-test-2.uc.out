---
test: run suites/task-1/cli-test-2.uc
---
41
3

exit: 0
